projects/showcase/src/app/shared/layout/layout.component.ts

```diff
@@ -13,17 +14,31 @@ import { MatToolbarModule } from '@angular/material/toolbar';
           <mat-icon svgIcon="ngx-maplibre-gl"></mat-icon>
         </a>
         <a mat-button routerLink="/"> ngx-maplibre-gl </a>
-        <a mat-button routerLink="/demo"> Demo </a>
-        <a mat-button routerLink="/doc"> Documentation </a>
+        <div class="menu-items">
+          <a mat-button routerLink="/demo"> Demo </a>
+          <a mat-button routerLink="/doc"> Documentation </a>
+        </div>
       </div>
       <div>
         <div id="layout-right-custom-items"></div>
-        <a mat-button href="https://github.com/maplibre/ngx-maplibre-gl">
-          Github
-        </a>
+        <!-- <a mat-button href="https://github.com/maplibre/ngx-maplibre-gl"> </a> -->
         <a mat-icon-button href="https://github.com/maplibre/ngx-maplibre-gl">
           <mat-icon svgIcon="github"></mat-icon>
         </a>
+        <button
+          class="mobile-menu-button"
+          mat-icon-button
+          type="button"
+          aria-label="Mobile menu"
+          [matMenuTriggerFor]="mobileMenu"
+        >
+          <mat-icon>more_vert</mat-icon>
```

Comment:
```suggestion
          <mat-icon>menu</mat-icon>
```

Reply:
`menu` icon is already used for the sidenav button on the demo pages. 

It would look like this if we use it for the mobile menu too:
![CleanShot 2023-12-18 at 12 28 42](https://github.com/maplibre/ngx-maplibre-gl/assets/8985933/674b8ba4-7ec9-4ae4-be55-17b7f9ea3f1b)

Reply:
Oh, right, my bad.
Can you increase its size like the right one is?
Also I think we can skip the text (ngx-maplibre-gl) in mobile devices, can't we?

Reply:
![CleanShot 2023-12-18 at 12 39 48](https://github.com/maplibre/ngx-maplibre-gl/assets/8985933/18def8cc-be85-407f-b641-fccc40ce88f4)

Reply:
Nice!